R/pkg/R/mllib.R

```diff
@@ -1218,11 +1218,11 @@ setMethod("spark.survreg", signature(data = "SparkDataFrame", formula = "formula
 #' @param data A SparkDataFrame for training.
 #' @param features Features column name. Either libSVM-format column or character-format column is
 #'        valid.
-#' @param k Number of topics.
-#' @param maxIter Maximum iterations.
+#' @param k Number of topics, default is 10.
```

Comment:
we discussed this earlier - we should avoid having default value in the doc since it's in the function signature already, unless there is additional information or explanation we are adding.

ditto below.